.travis.yml

```diff
@@ -26,9 +26,10 @@ matrix:
       env: SYMFONY_VERSION='^3.4'
     - php: 7.3
       env: SYMFONY_VERSION='^4.2'
-
-before_install:
-  - phpenv config-rm xdebug.ini
+    - php: 7.4snapshot
+      env: SYMFONY_VERSION='^3.4'
```

Comment:
no need for these 2 jobs testing 7.4 against specific SF versions. A single job running PHP 7.4 is enough (and we only need a single job forcing specifc SF LTS versions)